Munich/Museum5Kont/MfVK79300820.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
```

Comment:
This file here is due to the branch confusion. Can you discard this change on this branch?